src/validate.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+function validate(text, toCase) {
```

Comment:
The function name 'validate' is too generic, consider renaming it to 'validateTextConversion' or smth else for better clarity.